pallets/airdrop/src/tests/user_claim.rs

```diff
@@ -0,0 +1,419 @@
+use super::prelude::*;
+use crate::tests::UserClaimTestCase;
+use frame_support::traits::{Currency, LockableCurrency, ReservableCurrency};
```

Comment:
can we please remove `LockableCurrency` and  `ReservableCurrency` since they aren't used anywhere in this file?